src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,35 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img data-cy="MovieImage" src={movie.imgUrl} alt="Film logo" />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
```

Comment:
```suggestion
            <img src="images/imdb-logo.jpeg" alt={movie.title} />
```
Consider adding an informative alt attribute for better accessibility